packages/go_router/lib/src/route.dart

```diff
@@ -275,6 +275,7 @@ class GoRoute extends RouteBase {
     super.parentNavigatorKey,
     super.redirect,
     this.onExit,
+    this.caseSensitive = false,
```

Comment:
@chunhtai, from your comment https://github.com/flutter/flutter/issues/162487#issuecomment-2641358762, 

> based on https://www.w3.org/TR/WD-html40-970708/htmlweb.html, the URL should be case sensitive

I prefer to set the default to `true, ' but that would be a breaking change. I'm fine writing a migration doc

What do you prefer me to do?

Reply:
I agree we should probably default it to true to provide the correct behavior by default, but yeah we should bump major version and a breaking change doc if we do that

Reply:
Ok ! Will do !